### ISSUE TITLE
can't turn page with go to the corrner in pdf files

### ISSUE THREAD

**Issue by Monirzadeh** [koreader/koreader#7811]
* KOReader version:2021.5-47
* Device: Kindle PW

#### Issue
when you start highlighting and go to the corner of screen, koreader turn page automatically- so you can continue  highlight document in epub files. but it is not work in pdf files

#### Steps to reproduce
1. open a pdf file with koreader
2. long press in word to active highlight mode
3. swap to the corner of screen
4. page turn not work

**Comment by NiLuJe** [comment]
That's indeed not implemented in PDF engines, IIRC.

**Comment by Monirzadeh** [comment]
> IIRC

is there specific reason?

**Comment by NiLuJe** [comment]
PDF is hell.

**Comment by NiLuJe** [comment]
Besides that, the PDF backend is slightly nightmarish to maintain, requires actually understanding the PDF spec for some of the gnarliest aspects, was written ages ago, we don't currently have any maintainers interested in more than a quick easy fix here and there, and it just "mostly" works as-is.

TL;DR: Every issue more-or-less related to PDF will get a "help-wanted" stamp.
